### PR TITLE
Refocus collector runnable in PRD - sending Trust1 samples to staging

### PR DESCRIPTION
**Pull request by pallavi2209**

### CODE REVIEW

File: src/config/utils.js [hunk 1]
```diff
@@ -89,7 +89,16 @@ function init(reg) {
     }
   }
 
-  conf.registry = r;
+  conf.registry = {
```
Comment (iamigo) [inline src/config/utils.js]: uh oh! not checking this into source control!

File: src/heartbeat/utils.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ function addGenerator(res) {
 
   // Get a fresh copy of collector config
   const config = configModule.getConfig();
-  const refocusInstance = config.refocusInstance;
+  const refocusInstance = config.registry.refocusInstances.sandbox;
```
Comment (iamigo) [inline src/heartbeat/utils.js]: not checking this into source control

File: src/remoteCollection/collect.js [hunk 3]
```diff
@@ -16,6 +16,8 @@ const urlUtils = require('./urlUtils');
 const errors = require('../errors');
 const constants = require('../constants');
 const RefocusCollectorEval = require('@salesforce/refocus-collector-eval');
+require('superagent-proxy')(request);
+const proxy = 'http://public0-proxy1-0-prd.data.sfdc.net:8080';
```
Comment (iamigo) [inline src/remoteCollection/collect.js]: we're going to have to make this an optional command line arg

**Comment by pallavi2209** [comment]
Lots of changes since I started working on this - never rebased - let me update the PR with new changes.
Btw, this PR is only POC for trust1 sample generater template and running collector to push trust1 samples into staging. So, I don't think we need to merge anything here. just placing this branch here for reference.

**Comment by pallavi2209** [comment]
Closing this PR, will create new one with latest changes.

**Pull request closed without merging**